### PR TITLE
Added an example for info() method

### PR DESCRIPTION
**Pull request by javiereguiluz**
| Q | A |
| --- | --- |
| Doc fix? | no |
| New docs? | yes |
| Applies to | all |
| Fixed tickets | - |

This PR just rebases the work made by @OskarStark in #5632.

### CODE REVIEW

File: components/config/definition.rst [hunk 1]
```diff
@@ -425,9 +425,34 @@ All options can be documented using the
 :method:`Symfony\\Component\\Config\\Definition\\Builder\\NodeDefinition::info`
 method.
 
+.. code-block:: php
+
+    $rootNode
+        ->children()
+            ->inetegerNode('entries_per_page')
+                ->info('This value is only used for the search results page.')
+                ->devaultValue(25)
```
Comment (OskarStark) [inline components/config/definition.rst]: deFaultValue

Comment (javiereguiluz) [inline components/config/definition.rst]: Fixed. Thanks.

File: components/config/definition.rst [hunk 2]
```diff
@@ -425,9 +425,34 @@ All options can be documented using the
 :method:`Symfony\\Component\\Config\\Definition\\Builder\\NodeDefinition::info`
 method.
 
+.. code-block:: php
+
+    $rootNode
+        ->children()
+            ->inetegerNode('entries_per_page')
```
Comment (OskarStark) [inline components/config/definition.rst]: integerNode has a typo :smile:

Comment (javiereguiluz) [inline components/config/definition.rst]: This clearly proves that I didn't review anything when creating this PR. It's fixed now. Thanks.

Comment (OskarStark) [inline components/config/definition.rst]: :+1:

**Comment by xabbuh** [comment]
:+1:

**Comment by OskarStark** [comment]
This should be merged now IMO

**Comment by WouterJ** [comment]
Thanks @javiereguiluz and @OskarStark! I've merged this PR now.

Fyi, I've removed the XML example in the 2.3 docs, as the Xml dumper was introduced in Symfony 2.4. I've made sure it's there in the 2.7+ docs.

**Comment by OskarStark** [comment]
:+1:

**Pull request merged**